lib/Sema/CSSimplify.cpp

```diff
@@ -10856,6 +10862,10 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyFixConstraint(
   case FixKind::AllowCheckedCastCoercibleOptionalType:
   case FixKind::AllowUnsupportedRuntimeCheckedCast:
   case FixKind::AllowAlwaysSucceedCheckedCast:
+  case FixKind::AllowWrappedValueMismatch: {
+    return recordFix(fix) ? SolutionKind::Error : SolutionKind::Solved;
+  }
+
   case FixKind::AllowInvalidStaticMemberRefOnProtocolMetatype: {
```

Comment:
Nit: Could this could be joined? Since logic is the same :) 
```suggestion
  case FixKind::AllowWrappedValueMismatch: 
  case FixKind::AllowInvalidStaticMemberRefOnProtocolMetatype: {
```

Reply:
Oh I see, sorry for misunderstanding your last comment, that makes sense!